src/components/client-list/client-list.tsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+import { up } from '../breakpoint/breakpoint';
+import { Link } from 'gatsby';
+import { theme } from '../layout/theme';
+
+interface ClientListProps {
+  clients: [ClientListEntryProps];
+}
+
+interface ClientListEntryProps {
+  title: string;
+  timestamp: string;
+  link: string;
+}
+
+const linkStyles = css`
+  color: ${theme.palette.text.darkLinkColor.default};
+  font-size: 20px;
+  line-height: 110%;
+  text-decoration: none;
+
+  &:hover {
+    color: ${theme.palette.text.darkLinkColor.hover};
+  }
+`;
+
+const Wrapper = styled.div`
+  width: 100vw;
```

Comment:
It looks like `100vw` ignores the scroll bar. Don't ask me why 🙈 If you make the width `calc(100% + 32px)` it also works with the scroll bar.

![image](https://user-images.githubusercontent.com/7814926/82785966-e6d6a900-9e63-11ea-8015-7aac16c56010.png)

Reply:
Oh, interesting. I will check that out.
Is that the case for every browser/OS? Will investigate. 🤓

Reply:
Ah, you are right, on the browsers mobile view it works. But if you use the desktop view and just make the window smaller, there is now a vertical scroll bar. This is becuase of the `width: 100vw`.

<details>
 <summary>width: 100vw</summary>
<img width="914" alt="CleanShot 2020-05-26 at 09 07 41@2x" src="https://user-images.githubusercontent.com/7814926/82870860-ef48e580-9f30-11ea-83e1-796734c8b5df.png">
</details>

<details>
 <summary>width: calc(100% + 32px)</summary>
<img width="911" alt="CleanShot 2020-05-26 at 09 06 41@2x" src="https://user-images.githubusercontent.com/7814926/82870900-fec82e80-9f30-11ea-938b-8d0f1a933404.png">
</details>

Reply:
Where does this calculation come from?
To be honest a width of `100vw` is unlikely to cause a horizontal scroll bar.

Reply:
i don't see the horizontal scroll bar either :)

Reply:
i retract my previous statement 😅: i get the horizontal scroll bar at smaller widths, while using a mouse (not trackpad)

<img width="521" alt="Screenshot 2020-05-29 at 12 24 21" src="https://user-images.githubusercontent.com/31735645/83249755-5b348380-a1a7-11ea-80a8-a93786b3c910.png">

Reply:
Very crazy behaviour: When using a mouse, Chrome treats scroll bars differently then when using trackpad. There are also a bunch of articles online about this: https://jaketrent.com/post/100vw-not-scrollbar-aware/

I would recommend using `width: calc(100% + 32px)` as this "just works".

Reply:
But the calculation seams really arbitrary.. where does this come from?

Reply:
100% = grid with, 32px = gap on the left + gap on the right (16px each)

Reply:
Done.